lib/uksched/include/uk/thread.h

```diff
@@ -54,45 +54,74 @@ typedef void (*uk_thread_fn1_t)(void *) __noreturn;
 typedef void (*uk_thread_fn2_t)(void *, void *) __noreturn;
 
 struct uk_thread {
-	struct ukarch_ctx    ctx;	/**< Architecture context */
-	struct ukarch_ectx *ectx;	/**< Extended context (FPU, VPU, ...) */
-	uintptr_t           tlsp;	/**< Current active TLS pointer */
-	__uptr            uktlsp;	/**< Unikraft TLS pointer */
+	/* Architecture context */
+	struct ukarch_ctx    ctx;
+	/* Extended context (FPU, VPU, ...) */
+	struct ukarch_ectx *ectx;
+	/**< Current active TLS pointer */
+	uintptr_t tlsp;
+	/* Unikraft TLS pointer */
+	__uptr uktlsp;
 
+	/* Tail queue maintained by the scheduler implementation */
 	UK_TAILQ_ENTRY(struct uk_thread) queue;
+
+	/* Flags for capturing the thread's state */
```

Comment:
```suggestion
	/* Flags for capturing the thread state */
```
Detail possible values of the flags.